systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -503,6 +607,26 @@ void testSecretsWithKafkaConnectWithTlsAndTlsClientAuthentication() {
     }
 
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Test validating Kafka Connect with TLS and SCRAM-SHA authentication along with associated resources setup and verification."),
+        steps = {
+            @Step(value = "Initialize test storage and resource manager", expected = "Instances created successfully"),
```

Comment:
Isn't resource manager initialized before? Also, I guess that it's not needed to mention it there (as it doesn't have much value in the whole test steps IMHO).

Reply:
True updated